src/vue/pages/Assets.vue

```diff
@@ -98,7 +101,8 @@ export default {
 
   async created () {
     await this.loadMyBusinesses()
-    if (this.isAccountCorporate) {
+
+    if (this.isAccountCorporate && this.isAccountHaveBusiness !== undefined) {
```

Comment:
If you write isAccountHaveBusiness correct, this check will not be needed

Reply:
changed